### ISSUE TITLE
Integrate Have I been pwned breach API (needs to be asynchronous)

### ISSUE THREAD

**Issue by maruthi-adithya** [snift/snift-api#40]
A security breach -> Poor Security practices in the past. Need to add this to final score

**Comment by shrirambalaji** [comment]
Yeah, mozilla already does this. If a website has been breached before, Firefox shows a pop-up stating there was a breach, how long ago the breach happened, and details of the records compromised if any.

**Comment by maruthi-adithya** [comment]
Yes @Shriram-Balaji , we ll add it too

**Comment by maruthi-adithya** [comment]
@Shriram-Balaji Instead of hitting this from the backend, why don't we hit it from the frontend directly. Anyways, it is not our data plus we don't need to store this. Let me know your thoughts on this
